website/docs/UserEvent.md

```diff
@@ -0,0 +1,30 @@
+---
+id: user-event
+title: User Event
+---
+
+### Table of contents
+
+- [`userEvent.setup`](#usereventsetup)
+  - [Options](#options)
+
+
+## `userEvent.setup`
+
+```ts
+userEvent.setup(options?: {
+  delay: number;
+  advanceTimers: (delay: number) => Promise<void> | void;
+})
+```
+
+Example
+```ts
+const user = userEvent.setup();
```

Comment:
I'm not a huge fan of having to call the setup function to be able to trigger events. Why not have a config like the one we already have ? User event config could even be added to the existing global config maybe. I think having to call the setup function in each test makes the API not as nice to use and I'd prefer to have a global config and be able then to customize options at events level, very much like what is currently available for queries

Reply:
This is the current [User Event](https://testing-library.com/docs/user-event/intro/) API, v14. In the previous version, v13 you didn't have to create that `user` object.

Although I would prefer to have simpler API (`userEvent.press()`), imo there is high value in matching the current RLT's API, as we try to do for all possible RTL API, which means using `userEvent.setup()`.

Reply:
Oh it seems to be used also to carry state between events, now I understand better why they chose this API. It doesn't look very useful in our case for now but if they opted for this on react side it may be the best long term option here as well. Also I agree it is good to have the same API so I'm onboard with this